### DISCUSSION TITLE
Where is the best place to begin learning to use ControlNet and the openpose editor?

### DISCUSSION THREAD

**Discussion by realdreamachine** [lllyasviel/stable-diffusion-webui-forge#1987] [Q&A]
Been using stable diffusion webui forge for a while now but only for generation, inpainting and upscaling. Most of the other features remain a mystery I don't touch. Does anyone recommend any particular step-by-step tutorial, as simply explained as possible for beginners with ControlNet? Thanks.

**Comment by dongxiat** [comment]
> Been using stable diffusion webui forge for a while now but only for generation, inpainting and upscaling. Most of the other features remain a mystery I don't touch. Does anyone recommend any particular step-by-step tutorial, as simply explained as possible for beginners with ControlNet? Thanks.

https://github.com/Haoming02/All-in-One-Stable-Diffusion-Guide

**Reply by realdreamachine** [reply]
Thank you. Think I'll forget openpose _and_ controlnet.
